code-challenges.js

```diff
@@ -31,24 +38,66 @@ const temp2 = 350
 const temp3 = 212
 // Expected output: "212 is at boiling point"
 
+const temperature =(temp) => {
+    if (temp > 212) {
+        return `${temp} is above boiling point`
+    } else if (temp === 212) {
+        return `${temp} is at boiling point`
+    } else {
+        return `${temp} is below boiling point`
+    }
+}
+console.log(temperature(temp1));
+console.log(temperature(temp2));
+console.log(temperature(temp3));
+
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Describe your process:
+//Create a variable to combine the two arrays into a new array
```

Comment:
I would really like to see you include any built-in methods you are considering using to achieve that result.